packages/react-components/src/components/RichTextEditor/RichTextSendBox.tsx

```diff
@@ -178,6 +190,19 @@ export interface RichTextSendBoxProps {
    * Optional callback called when user is typing
    */
   onTyping?: () => Promise<void>;
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  /**
+   * Optional callback to upload an inline image in the rich text editor.
+   * @beta
+   */
+  onUploadImage?: (image: string, fileName: string) => void;
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  /**
+   * Optional array of type {@link AttachmentMetadataInProgress}
+   * to render inline images being uploaded in the RichTextSendBox.
+   * @beta
+   */
+  uploadInlineImages?: AttachmentMetadataInProgress[];
```

Comment:
Should it be `inlineImages` to be consistent with `attachments` prop?

Reply:
> Should it be `inlineImages` to be consistent with `attachments` prop?

Was thinking of that but that could confuse people with the inlineImages for displaying in the messageThread component. Open to other naming suggestions

Reply:
Should we add inlineImages for displaying in the messageThread to message props instead of MessageThread itself?

Reply:
`imagesPendingUpload` 
`imageUploadsInProgress`
?